dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -51,6 +51,55 @@ import {
 import { space } from '@guardian/src-foundations';
 import { ContainerLayout } from '../components/ContainerLayout';
 
+const HeadlineGrid = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			/* IE Fallback */
```

Comment:
out of interest (because I'm not sure) do we run tests across different browsers?

Reply:
Good question! @oliverlloyd?

Reply:
Yes and no.

There are two places where we run tests in browsers, Cypress and Chromatic.

We, Dotcom, only use Chrome for Chromatic tests but other teams run snapshots on Chrome + Firefox + IE
We investigated running Cypress over different browsers but haven't moved forward with this and I don't think any other teams have

We _do_ do manual testing using BrowserStack for specific issues and development but this isn't automated.

In general, whilst browser support is critically important and all out features must work over all [our supported browsers](https://www.theguardian.com/help/recommended-browsers), these days there just aren't as many cross browser issues as there used to be so the trades offs for automation are not quite the same